gugel-project-2/App.js

```diff
@@ -0,0 +1,90 @@
+// In App.js in a new project
+import * as React from 'react';
+import { View, FlatList, ScrollView, Text, Button, SectionList, StyleSheet, TextInput, TouchableWithoutFeedback } from 'react-native';
+import {List, ListItem, SearchBar } from "react-native-elements";
+import { NavigationContainer, useNavigation, withNavigation } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+
+let movieList = [{title: 'Movie A', date: 1997, key: '1'}, {title: 'Movie B', date: 2000, key: '2'}, {title: 'Movie C', date: 1942, key:'3'}]
+const styles = StyleSheet.create({
+  title: {
+    fontSize: 48
+  },
+})  
+let ogMovieList = movieList
+let itemData = ''
+let textData = ''
+let newData = []
+
+let searchFilterFunction = (newData) => {    
+  setState({newData})
+}
```

Comment:
We don't need to implement this function, the useState hook should automatically update our state for us